assignments/prototypes.js

```diff
@@ -32,7 +50,48 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+function Humanoid(info) {
+  this.team = info.team;
+  this.weapons = info.weapons;
+  this.language = info.language;
+  CharacterStats.call(this, info)
+}
+
+Humanoid.prototype = Object.create(CharacterStats.prototype)
+
+Humanoid.prototype.greet = function() {
+  return `${this.name} offers a greeting in ${this.language}`
+}
+
+function Hero(ats) {
+  Humanoid.call(this, ats);
+}
+Hero.prototype = Object.create(Humanoid.prototype);
+
+Hero.prototype.attack = function(target, value) {
+  let damage = target.healthPoints - value;
```

Comment:
"Damage" is a little confusing here since "value" is the actual damage done, just minor semantic nuance